### PR TITLE
Readme.md: Heading capitalization and some cleanup

### PR DESCRIPTION
**Pull request by xmr-eric**
Most headings capitalized every word. A few did not. Now all do.

### CODE REVIEW

**Comment by moneromooo-monero** [comment]
I'd have thouhgt it'd be best to normalize NOT having capitals in the middle of sentences... We're not marketers :/

**Comment by xmr-eric** [comment]
@moneromooo-monero
I totally agree. Was just following suit for what had already existed. I would prefer just the initial word of every heading be capitalized.

That said, the Bitcoin github capitalizes every word like we do. So there is precedent to keep things this way.

Shall I switch the readme to just having the first word capitalized?

**Comment by moneromooo-monero** [comment]
Meh, it's a fairly common way to do it too, so I don't really care all that much. The bitcoin github isn't a particularly authoritative model for something like this fwiw.

**Comment by xmr-eric** [comment]
Edited with just first word capitalized. Let me know what you think.

**Comment by xmr-eric** [comment]
A few suggestions:

1. The "vulnerability response process" is listed twice, once at the top and another time under the Contributing header. Can one of them be removed?

2. The header "Monero software updates and network consensus protocol upgrades (hard fork schedule)" seems like it could be shortened. What about "Monero network protocol upgrades (hard fork schedule)"? IE: removing the term "software updates" would shorten the whole thing a bit.

**Comment by moneromooo-monero** [comment]
I personally prefer the new cap format, thanks :)

The second VRP paragraph seems entirely redundant indeed.

The long title is gingeropolous' construction, convince him :)

**Comment by Gingeropolous** [comment]
I don't care what the title is, as long as its just not "hard fork". I like the current title, obviously, because it explains things better than "hard fork".

perhaps it can just be "Network Consensus Protocol Upgrade "

and we can move "Software updates" into the title below.

**Comment by xmr-eric** [comment]
Edited. Hopefully this looks good.

File: README.md [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 - Forum: [forum.getmonero.org](https://forum.getmonero.org)
 - Mail: [dev@getmonero.org](mailto:dev@getmonero.org)
 - GitHub: [https://github.com/monero-project/monero](https://github.com/monero-project/monero)
 - IRC: [#monero-dev on Freenode](http://webchat.freenode.net/?randomnick=1&channels=%23monero-dev&prompt=1&uio=d4)
 
-## Vulnerability Response
+## Vulnerability response
 
-- Our [Vulnerability Response Process](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md) encourages responsible disclosure
+- Our [vulnerability response process](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md) encourages responsible disclosure
```
Comment (anonimal) [inline README.md]: The VRP (and the adjectives which define the process) are defined as a *proper noun*. As such, the caps should stay.

**Comment by NanoAkron** [comment]
I don’t think there has been enough discussion to merge this one. We need at least 50 comments for a change this big.

**Comment by danrmiller** [comment]
I don't really care, but I don't like "Network Consensus Protocol Upgrade" over hard fork, because the network protocol doesn't seem to affect consensus at all.

File: README.md (hunk 1 above)
Comment (xmr-eric) [reply README.md]: Fixed

**Comment by Gingeropolous** [comment]
well it aint my fault bitcoin poisoned the notion of a hard fork. 

aight then, Blockchain Inclusion Consensus Upgrade ... because it does change what is considered valid to be entered into the block. 

whose riding dem bikes?

**Comment by danrmiller** [comment]
I like "hard fork".

**Comment by xmr-eric** [comment]
I prefer the heading be similar: Hard fork schedule

**Comment by lh1008** [comment]
I prefer hard fork.. :)

**Comment by Gingeropolous** [comment]
aight, then you guys get to do all the explaining on reddit :)

**Comment by dEBRUYNE-1** [comment]
Fwiw, I used `Scheduled Protocol Upgrade` last time. I am not in favor of using the hard fork, as it might cause a lot of uproar.

**Comment by xmr-eric** [comment]
After some discussion, people seem to agree on "Scheduled mandatory software upgrades" as the ideal heading for the section about scheduled hard forks. See recent commit and this discussion: https://paste.fedoraproject.org/paste/B~KljNfTRe1i2XFS0BgMFA

**Pull request merged**